pkg/vsphere/session/errors.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2016 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package session
+
+import "fmt"
```

Comment:
We need exported errors to access them outside the `session` pkg in `cmd/vicadmin/server.go`. Since these errors are session-specific, I put them in the `session` pkg. Should we move them to `pkg/errors` anyway?